### PR TITLE
implement github action

### PR DESCRIPTION
**Pull request by mifi**

### CODE REVIEW

**Review by juliangruber — APPROVED** [review]

**Comment by juliangruber** [comment]
Only concern is that these tests run on the GitHub provided tests runners, and one run took ~1 minute, however I don't expect much action here so it should be fine

**Comment by mifi** [comment]
What's the issue with github test runners? it's a public project, so it's unlimited free minutes I think?

**Comment by juliangruber** [comment]
True sorry I forgot that this is a public project

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+
+      - run: yarn
+      - run: npm run lint
```
Comment (kvz) [inline .github/workflows/test.yml:17]:
```suggestion
      - run: yarn lint
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      - run: npm run test
```
Comment (kvz) [inline .github/workflows/test.yml:18]:
```suggestion
      - run: yarn test
```

**Review by kvz — APPROVED** [review]

**Pull request merged**